httpserver/http_connection.cpp

```diff
@@ -40,6 +40,10 @@ namespace loki {
 
 constexpr auto SESSION_TIME_LIMIT = std::chrono::seconds(30);
 
+// Note: on the client side the limit is different
+// as it is not encrypted/encoded there yet
+constexpr size_t MAX_MESSAGE_BODY = 3100;
```

Comment:
Can the comment explain where `3100` comes from?

Reply:
sure, updated